src/ts/swagger/index.ts

```diff
@@ -29,6 +33,42 @@ const onWindowLoad = () => {
   };
 
   browser.runtime.onMessage.addListener(messageHandler);
+
+  // Does not seem to need `bind`
+  const setSpec = editor.unmockActions.setSpec;
+  await fillFromTab(setSpec);
+};
+
+const fillFromTab = async (updateSpec: (content: string) => void) => {
+  const result = (await storage.getTabInfo()) || {};
+  const tabIdOrUndefined = result.tabId;
+  if (!tabIdOrUndefined) {
```

Comment:
Can `tabId` be 0? Won't this fail then?

Reply:
Good catch, no idea if it could be zero but why not.